packages/tkg-clusterclass/bundle/config/packageinstalls/tkg-infra-clusterclass-service-account.yaml

```diff
@@ -29,41 +29,6 @@ rules:
       - get
       - list
       - delete
-      - watch
```

Comment:
Why these rules are not required?

Reply:
It was introduced by https://github.com/vmware-tanzu/tanzu-framework/pull/3683 and should be cleaned up by https://github.com/vmware-tanzu/tanzu-framework/pull/3849.